images/capi/packer/qemu/packer.json

```diff
@@ -48,7 +44,7 @@
   },
   "builders": [
     {
-      "name": "{{ user `build_name`}}",
+      "name": "flatcar-stable-2512.2.1",
```

Comment:
Is this something we'll need to bump regularly upstream?

Reply:
Good point. I will soon remove the suffix `-2512.2.1`.
To do that, I would also need to update `images/capi/hack/image-build-flatcar.sh`, so it does not depend on a specific Flatcar version.

Reply:
Done